src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/RegressionTest.java

```diff
@@ -122,6 +127,109 @@ public void testSelectIntoUpdateCount() throws SQLException {
         if (null != con)
             con.close();
     }
+   
+    /**
+     * Tests update query
+     * 
+     * @throws SQLException
+     */
+    @Test
+    public void testUpdateQuery() throws SQLException {
+        SQLServerConnection con = (SQLServerConnection) DriverManager.getConnection(connectionString);
+        String sql;
+        PreparedStatement pstmt = null;
+        JDBCType[] targets = {JDBCType.INTEGER, JDBCType.SMALLINT};
```

Comment:
also here changing to int[] targets = {java.sql.Types.INTEGER, java.sql.Types.SMALLINT}; should solve issue for java 7 issue.